src/datasources/balances-api/zerion-balances-api.service.ts

```diff
@@ -222,9 +222,9 @@ export class ZerionBalancesApi implements IBalancesApi {
           throw Error(
             `Zerion error: ${chainName} implementation not found for balance ${zb.id}`,
           );
-        const { value } = zb.attributes;
+        const { value, price } = zb.attributes;
         const fiatBalance = value ? getNumberString(value) : null;
-        const fiatConversion = getNumberString(zb.attributes.price);
+        const fiatConversion = price ? getNumberString(price) : null;
```

Comment:
`attributes.price` is not marked as required in the [Zerion API docs](https://developers.zerion.io/reference/listwalletpositions), so I've changed this to reflect the possibility of the field being `undefined`.